elastic/kibana
# [Infra][ObsUX] Hosts & Container Logs only overview

## Summary

Enables a logs only overview for hosts & containers. Disables the metrics tab as there's no data incoming for metrics, and provides Logs charts on the overview page detailing the Log Rate (all logs generated) and Log Error Rate (all recorded errors).

https://github.com/user-attachments/assets/ced14b6d-dd08-4514-9066-6c02c62d5ff8

Closes #201752

## How to test

This is tested using synthtrace data, loading the scenario below:

```
node scripts/synthtrace traces_logs_entities.ts --live
```

This loads a logs-only host, though if other scenarios contain logs only containers, feel free to use those as well.

* Go to Inventory page. Click on a host or container.
* If it is a logs only host/container, no metrics tab should be shown. Metrics KPI charts should be replaced with Logs KPI charts (Log Rate and Log Error Rate).
* If the host/container contains metrics, the metrics tab should be visible and the normal Metrics KPI charts should be present.
